server/backend/users/tests/test_serializers.py

```
@@ -0,0 +1,34 @@
+from users.models import Instructor
+from users.serializers import InstructorSerializer
+
+User = get_user_model()
+
+@pytest.fixture
+def create_instructor():
+    user = User.objects.create_user(username="john_doe", password="password123", role="instructor")
+    return Instructor.objects.create(user=user, designation="Professor", university="ABC University")
+
+@pytest.mark.django_db
+def test_instructor_serializer(create_instructor):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_instructor_serializer`

Add return type annotation: `None`

(ANN201)